main/EventSystem.cpp

```diff
@@ -184,12 +184,26 @@ void CEventSystem::SetEnabled(const bool bEnabled)
 
 void CEventSystem::LoadEvents()
 {
-	std::string dzv_Dir,s;
+	std::string dzv_Dir, dzv_scripts, s;
 #ifdef WIN32
 	dzv_Dir = szUserDataFolder + "scripts\\dzVents\\generated_scripts\\";
+	dzv_scripts = szUserDataFolder + "scripts\\dzVents\\";
 #else
 	dzv_Dir = szUserDataFolder + "scripts/dzVents/generated_scripts/";
+	dzv_scripts = szUserDataFolder + "scripts/dzVents/";
 #endif
+	const std::string // remove obsolete dirs
+	dzv_rm_Dir1 = dzv_scripts + "runtime",
+	dzv_rm_Dir2 = dzv_scripts + "documentation";
+
+	if ((file_exist(dzv_rm_Dir1.c_str()) || file_exist(dzv_rm_Dir2.c_str())) &&
+		!szUserDataFolder.empty())
+	{
+		std::string errorPath;
+		if (int returncode = RemoveDir(dzv_rm_Dir1 + "|" + dzv_rm_Dir2, errorPath))
```

Comment:
Will this be executed every time 'LoadEvents' is executed ?
Normally we place this kind of 'patches' inside SQLHelper.cpp and increase the database version number

Reply:
Better indeed, I thought it was better to keep all in eventsystem, but with moving the removedir function to helper, we should do it this way. Will change.